### PR TITLE
[Messenger] Add reference to PHP docs for SSL context options

### PR DESCRIPTION
**Pull request by phansys**
Follows https://github.com/symfony/symfony-docs/pull/20094#issuecomment-2338271547.

### CODE REVIEW

File: messenger.rst [hunk 1]
```diff
@@ -1694,7 +1694,7 @@ redis_sentinel           support is disabled
 ssl                      Map of TLS options.                    null
 =======================  =====================================  =================================
 
-The ``ssl`` option can be used to change requirements for the TLS channel, e.g. in tests:
+The ``ssl`` option can be used to provide SSL context options (`php.net/context.ssl`_) for the TLS channel, e.g. in tests:
```
Comment (MrYamous) [inline messenger.rst:1697]:
```suggestion
The ``ssl`` option can be used to provide `SSL context options`_ for the TLS channel, e.g. in tests:
```
WDYT about this for better link integration in text ?

Comment (phansys) [reply messenger.rst:1697]: I think we could use the proposed variant if this keeps the reference intact to the PHP docs.

image

See: https://github.com/symfony/symfony-docs/blob/f39f6576b458adf4ab04726066f921557a7041cb/components/cache/adapters/redis_adapter.rst?plain=1#L220

Comment (javiereguiluz) [reply messenger.rst:1697]: Yes, @MrYamous proposal is what we usually do in the docs. So, I changed this while merging. See https://github.com/symfony/symfony-docs/commit/714113ceff60c34a186783f6d4163875d1105bc7

**Comment by javiereguiluz** [comment]
Thanks Javier!

**Pull request merged**